src/App.js

```diff
@@ -1,23 +1,70 @@
-import logo from './logo.svg';
+// Modules
+import firebase from './firebase';
+import {useEffect, useState} from 'react';
+import {getDatabase, ref, onValue, push} from 'firebase/database';
+// Components
+import Header from "./Components/Header";
+import Cart from "./Components/Cart";
+// Assets
 import './App.css';
 
 function App() {
+  // products state for inventory
+  const [products, setProducts] = useState([]);
+  // cart state for cart items
+  const [cart, setCart] = useState([]);
+
+
+  useEffect(() => {
+    const database = getDatabase(firebase)
+    const dbRef = ref(database, "/inventory")
+    const cartRef = ref(database, "/cart")
+    // pulling inventory data
+    onValue(dbRef, (response) => {
+      const newState = [];
+      const data = response.val();
+      for (let key in data) {
+        newState.push(data[key]);
+      }
+      setProducts(newState);
+    })
+    // pulling cart data
```

Comment:
Really like the comments and the overall organization of yours js code. Everything is super easy to follow and understand!!